jackson2/pom.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+~   Licensed under the Apache License, Version 2.0 (the "License");
+~   you may not use this file except in compliance with the License.
+~   You may obtain a copy of the License at
+~
+~   http://www.apache.org/licenses/LICENSE-2.0
+~
+~   Unless required by applicable law or agreed to in writing, software
+~   distributed under the License is distributed on an "AS IS" BASIS,
+~   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+~   See the License for the specific language governing permissions and
+~   limitations under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>jdbi3-parent</artifactId>
+        <groupId>org.jdbi</groupId>
+        <version>3.6.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>jdbi3-jackson2</artifactId>
+    <name>jdbi3 jackson2</name>
+    <description>Jdbi Jackson 2.0 integration</description>
+    <url>http://jdbi.org/</url>
+
+    <properties>
+        <jackson.version>2.6.0</jackson.version>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.jdbi</groupId>
+            <artifactId>jdbi3-core</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-annotations</artifactId>
+            <version>${jackson.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-core</artifactId>
+            <version>${jackson.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-databind</artifactId>
+            <version>${jackson.version}</version>
```

Comment:
Should we manage this in the parent POM instead?

Reply:
I'd put it here since no other modules should ever depend on jackson, so there is no reason for them to inherit its management.  Do you think we should put it in the parent anyway?

Reply:
I think it's good practice to always put dependency versions in the parent regardless of usage location, just to avoid ever getting different versions of the same thing or spreading version management out over multiple places.

Reply:
Right, and that's exactly what I'd imagined here -- someday there will be a Jackson 3 and that will be a new `jackson3` module, and you don't want the `jackson2` dependency management creeping in.  Or someone (foolishly) decides to contribute a `jackson1` module.  So would you consider this such an exception?

Reply:
Well you could leave it in the parent because there's no (expectation of a) competing version module _yet_, or you could move it inside the module preemptively... The difference though is that with the former you'll notice the necessary change at dev time, while with the latter another jackson version might be added to another module for whatever reason and then there'll be conflicts _at runtime_ for users. It feels like opening the door to an (unlikely) unwelcome surprise. I would prefer it in the parent until there's a competing module, but it's not like I'm gonna make a big deal out of it if you feel strongly and make the decision to not do that ¯\\\_(ツ)\_/¯